plugins/command-line/prism-command-line.js

```diff
@@ -63,7 +63,10 @@ Prism.hooks.add('complete', function (env) {
 
 		if (!isNaN(outputStart) && !isNaN(outputEnd)) {
 			for (var j = outputStart; j <= outputEnd && j <= prompt.children.length; j++) {
-				prompt.children[j - 1].className = 'output';
+				var node = prompt.children[j - 1];
+				node.removeAttribute('data-user');
+				node.removeAttribute('data-host');
+				node.removeAttribute('data-prompt');
```

Comment:
In my opinion, the original code is cleaner and easier to read.  It's "nice" that it could be done in 1 line, but saving those 18 characters effectively adds 4 function calls and 3 string concatenations.